src/Kentico.Xperience.Lucene.Sample/appsettings.json

```diff
@@ -17,11 +17,12 @@
     "CMSConnectionString": ""
   },
   "CMSHashStringSalt": "195a79ed-c559-42d8-b801-5857fc07ab1c",
-
-  "CMSAdminClientModuleSettings": {
-    "kentico-xperience-integrations-lucene": {
-      "Mode": "Proxy",
-      "Port": 3009
-    }
-  }
+  "WebCrawlerBaseUrl": "http://localhost:41489/"
+  //// uncomment for client admin module development
```

Comment:
Yeah, this should probably go in User Secrets.
We can update the documentation in the `README.md` to explain this should be added, [in the Contributing section](https://github.com/Kentico/xperience-by-kentico-lucene#contributing) and then we can remove it from the `appsettings.json`.